server/types/validators.ts

```diff
@@ -0,0 +1,25 @@
+import { VolunteerDataSchema } from './types'
+import { Request, Response, NextFunction } from 'express';
+
+type ExpressValidator = (req: Request, res: Response, next: NextFunction) => void;
+
+const defaultValidator: ExpressValidator = (req: Request, res: Response, next: NextFunction) => {
+    next();
+}
+
+const volunteerDataValidator: ExpressValidator = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I think this validation pipeline could be abstracted out nicely, but that can come later!